docs/resources/Stage_Instance.md

```diff
@@ -0,0 +1,71 @@
+# Stage Instance Resource
+
+A _Stage Instance_ holds information about a live stage.
+
+## Definitions
+
+Below are some definitions related to stages.
+
+- **Liveness:** A Stage channel is considered _live_ when there is an associated stage instance. Conversely, a Stage channel is _not live_ when there is no associated stage instance.
```

Comment:
How can one know if there is an associated stage instance? There doesn't seem to be a documented endpoint or attribute to hint at this.

Reply:
This is a good callout. The answer right now for our clients is that we use the Gateway.

However, thinking about it more I'm not sure that bots want to keep state forever to handle this type of stuff. I'll see if we can expose a `GET /stage-instances/<channel_id>` endpoint next week.

Reply:
`GET /stage-instances/<channel_id>` has been implemented now.